src/tags/object/Video/HtxVideo.js

```diff
@@ -175,11 +170,11 @@ const HtxVideoView = ({ item }) => {
   }, [panMode, pan]);
 
   const zoomIn = useCallback(() => {
-    setZoom(clamp(zoom + 0.1, 0.25, 16));
+    setZoom(zoom + 0.1);
```

Comment:
since 0.1 is used in zoomIn and zoomOut it would be great to create a constant variable called something like `interval` or `zoomInterval` that we can reuse